Sources/Plasma/PubUtilLib/plAvatar/plPhysicalControllerCore.cpp

```diff
@@ -104,15 +104,15 @@ void plPhysicalControllerCore::IncrementAngle(float deltaAngle)
     fLocalRotation.NormalizeIfNeeded();
     fLocalRotation.GetAngleAxis(&angle, &axis);
     if (axis.fZ < 0)
-        angle = (2.0f * float(M_PI)) - angle; // axis is backwards, so reverse the angle too
+        angle = hsConstants::two_pi<float> - angle; // axis is backwards, so reverse the angle too
 
     angle += deltaAngle;
 
     // make sure we wrap around
     if (angle < 0.0f)
-        angle = (2.0f * float(M_PI)) + angle; // angle is -, so this works like a subtract
-    if (angle >= (2.0f * float(M_PI)))
-        angle = angle - (2.0f * float(M_PI));
+        angle = hsConstants::two_pi<float> + angle; // angle is -, so this works like a subtract
+    if (angle >= hsConstants::two_pi<float>)
+        angle = angle - hsConstants::two_pi<float>;
```

Comment:
```suggestion
        angle += hsConstants::two_pi<float>; // angle is -, so this works like a subtract
    if (angle >= hsConstants::two_pi<float>)
        angle -= hsConstants::two_pi<float>;
```

I feel like this is more readable, but it's no showstopper. I wish this could use something like `std::clamp` instead of all of these weird comparisons :/